### DISCUSSION TITLE
Are two identical useQuery calls unsupported?

### DISCUSSION THREAD

**Discussion by Mookiies** [urql-graphql/urql#1777] [Q&A] [answered]
I saw this comment that that says that two identical `useQuery` calls is an unsupported edge-case https://github.com/FormidableLabs/urql/discussions/1071#discussioncomment-106736. I saw that discussion was resolved, but is what was written about identical `useQuery`'s still true?

Is the correct way to leverage `useQuery` to only have one top level query and then distribute that data with prop-passing or in more complex and deeply nested cases with state-management?

**Comment by JoviDeCroock** [comment, marked as answer]
This should have been resolved, this issue used to present itself only with policies like `cache-and-network` though
